guides/k8s-microservices/src/main/paradox/index.md

```diff
@@ -0,0 +1,323 @@
+To do this, the first node is used as seed node and referenced by environment variables. Chirper defines `StatefulSet`
+resources for each of its services: `friendservice`, `activityservice`, `chirpservice`, and `web`
+
+##### Service
+
+A [Service](https://kubernetes.io/docs/concepts/services-networking/service/) provides the means to expose TCP and
+UDP ports to other pods within the cluster, and it integrates with DNS so that they can be discovered via a DNS SRV
+lookup. Chirper uses `Service` definitions so that its services can communicate with each other.
+
+##### Ingress
+
+An [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/) is a collection of rules that 
+allow external traffic to reach services running inside Kubernetes. This allows, for example, 
+requests to `/api/users` to be routed to the `friendservice` while requests for `/` are routed to `web`. It also gives 
+you a central place to terminate TLS and Chirper is configured to do this for you.
+
+Chirper is configured to use nginx as the ingress controller.
+
+### Lagom Service Locator
+
+Lagom makes use of a [Service Locator](https://www.lagomframework.com/documentation/1.3.x/java/ServiceLocator.html) to
+call other services within the system. Chirper uses the [service-locator-dns](https://github.com/typesafehub/service-locator-dns)
+project to provide a service locator that takes advantage of Kubernetes [Service Discovery](https://kubernetes.io/docs/concepts/services-networking/service/#discovering-services).
+
+Because the names of your service locators will not exactly match the DNS SRV address, `service-locator-dns` has
+the ability to translate addresses. Chirper uses this feature to ensure, for example, a service lookup for `friendservice`
+will be translated into a DNS SRV lookup for `_http-lagom-api._tcp.friendservice.default.svc.cluster.local`. This is 
+achieved with the following configuration in each service's `application.conf`:
+
+```
+service-locator-dns {
+  name-translators = [
+    {
+      "^_.+$" = "$0",
+      "^.*$" = "_http-lagom-api._tcp.$0.default.svc.cluster.local"
+    }
+  ]
+
+  srv-translators = [
+    {
+      "^_http-lagom-api[.]_tcp[.](.+)$" = "_http-lagom-api._http.$1",
+      "^.*$" = "$0"
+    }
+  ]
+}
+```
+
+_Refer to the various `application.conf` files in the Chirper repository for more details._
+
+### Installation script
+
+_Below, the install script in the Chirper repository is dissected. You can find it in its
+entirety at `deploy/k8s/minikube/scripts/install`_
+
+Now that all the resources required for deployment have been described, this guide will cover how to automate the process
+of deploying them to Kubernetes.
+
+Deploying Chirper requires the following actions:
+
+* Start up your Minikube
+* Deploy Cassandra
+* Build Chirper Docker images
+* Deploy Chirper
+* Deploy nginx
+* Open the service in your browser
+
```

Comment:
How can they run the example by just executing the install script? Did I miss that? I would add a level 2 heading "Running from the install script" and provide that info. and add a new level 2 heading, "Running the commands manually" and have the rest of the sections be at level 3 under that